docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -11,17 +11,24 @@ weight: 100
 
 This page explains how to install Grafana dependencies, download and install Grafana, get the service up and running on your Debian or Ubuntu system, and also describes the installation package details.
 
-## Repository migration (November 8th 2022)
+## Note on upgrading
 
-From that date, Grafana packages will be served from a new repository (<packages.grafana.com/deb/{product}> -> <apt.grafana.com>). The new repository serves, from a single APT configuration, all Grafana OSS products, as well as Grafana Enterprise.
+While the process for upgrading Grafana is very similar to installing Grafana, there are some key backup steps you should perform. Read [Upgrading Grafana]({{< relref "../../../upgrade-guide/" >}}) for tips and guidance on updating an existing installation.
 
-The old URLs will still work, serving the content from the new repository, but you may encounter warnings about some repository attributes changing (e.g. `Origin` and `Label`).
+## Download and install Grafana
 
-## Note on upgrading
+Complete the following steps to download and install Grafana:
 
-While the process for upgrading Grafana is very similar to installing Grafana, there are some key backup steps you should perform. Read [Upgrading Grafana]({{< relref "../../../upgrade-guide/" >}}) for tips and guidance on updating an existing installation.
+1. On the [Grafana download page](https://grafana.com/grafana/download), select the Grafana version you want to install.
+   - The most recent Grafana version is selected by default.
+   - The **Version** field displays only finished releases. If you want to install a beta version, click **Nightly Builds** and then select a version.
+1. Select an **Edition**.
+   - **Enterprise:** This is the recommended version. It is functionally identical to the open source version, but includes features you can unlock with a license if you so choose.
+   - **Open Source:** This version is functionally identical to the Enterprise version, but you will need to download the Enterprise version if you want Enterprise features.
+1. Depending on which system you are running, click the **Linux** or **ARM** tab on the download page.
+1. Copy and paste the code from the installation page into your command line and run.
 
-## 1. Download and install
+---
```

Comment:
@DanCech, from here forward through line 108, I'm unsure what, if any, of these instructions we should keep (they are very old). I want this topic to point to the download page where they select the release and copy and paste the instructions into the command line.

Reply:
We will need to retain the apt repo setup instructions, and in some ways it makes more sense for those to be here rather than on the download page

Reply:
Ok, good to know and I agree that we should keep them in the markdown. Thanks!